audio/music/superherosound.mp3.license

```diff
@@ -0,0 +1,9 @@
+Sound by Mishaal Kandapath
+
+# Attribution 4.0 International (CC BY 4.0)
+By using this file you accept that it won't be used for destructive purposes.

```

Comment:
adding this sentence modifies the license so it is not CC-BY and more. I can not guarantee that other people do not use it for those purposes.  Please remove the sentence to have a correct license in my eyes.